### PR TITLE
add support for specifying altroot/altversion in Bundle easyblock

### PR DESCRIPTION
**Pull request by boegel**
depends on ~~https://github.com/hpcugent/easybuild-framework/pull/1508~~, required for https://github.com/hpcugent/easybuild-easyconfigs/pull/2214

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

File: easybuild/easyblocks/generic/bundle.py [hunk 1]
```diff
@@ -33,15 +33,37 @@
...
+        extra_vars = {
+            'altroot': [None, "Software name of dependency to use to define $EBROOT for this bundle", CUSTOM],
+            'altversion': [None, "Software name of dependency to use to define $EBVERSION for this bundle", CUSTOM],
+        }
+        return EasyBlock.extra_options(extra_vars)
+
+    def __init__(self, *args, **kwargs):
+        """Initialize easyblock."""
+        super(Bundle, self).__init__(*args, **kwargs)
+        self.altroot, self.altversion = None, None
```
Comment (wpoely86) [inline easybuild/easyblocks/generic/bundle.py]: we never do this? split it out in two lines.

File: easybuild/easyblocks/generic/bundle.py [hunk 2]
```diff
@@ -33,15 +33,37 @@
...
+
     def configure_step(self):
-        """Do nothing."""
-        pass
+        """Collect altroot/altversion info."""
+        # pick up altroot/altversion, if they are defined
+        self.altroot = self.cfg['altroot']
+        if self.altroot:
+            self.altroot = get_software_root(self.altroot)
```
Comment (wpoely86) [inline easybuild/easyblocks/generic/bundle.py]: this is confusing. Do
```python
if self.cfg['altroot']:
  self.altroot = get_software_root(self.cfg['altroot'])
```

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**